h/services/oidc.py

```diff
@@ -91,21 +95,36 @@ def get_provider_unique_id(
 
         id_token = OIDCTokenResponseSchema().validate(token_response.json())["id_token"]
 
-        decoded_id_token = self._jwt_service.decode_oidc_idtoken(
+        decoded_idtoken = self._jwt_service.decode_oidc_idtoken(
             id_token, settings.keyset_url
         )
 
         try:
-            return decoded_id_token["sub"]
+            decoded_idtoken["sub"]
```

Comment:
Just preserving the existing behaviour where this raises `MissingSubError` if `sub` is not present. `sub` is a required field so this should never happen. It's not a bad idea to raise a specific exception if this unexpected condition ever does happen: it means we'll get a separate issue for it in Sentry (with its own separate Slack notification too).